Change Makefile to add to CFLAGS instead of replacing them.
--- a/Makefile
+++ b/Makefile
@@ -16,7 +16,7 @@
 
 CPPFLAGS+=	`pkg-config --cflags fontconfig x11 xft xinerama xrandr`
 
-CFLAGS=		-Wall -O2 -g -D_GNU_SOURCE
+CFLAGS+=	-Wall -O2 -g -D_GNU_SOURCE
 
 LDFLAGS+=	`pkg-config --libs fontconfig x11 xft xinerama xrandr`
 
